sky/tools/create_full_ios_framework.py

```diff
@@ -106,6 +106,46 @@ def main():
       args, dst, framework, arm64_framework, simulator_framework,
       simulator_x64_framework, simulator_arm64_framework
   )
+
+  # build extension safe frameworks
+  extension_safe_dst = os.path.join(dst, 'extension_safe')
```

Comment:
Here, I created a new subfolder under dst, extension safe frameworks are all moved into this subfolder.